core/chains/evm/gas/models.go

```diff
@@ -356,6 +369,7 @@ type GasEstimatorConfig interface {
 	TipCapMin() *assets.Wei
 	PriceMin() *assets.Wei
 	PriceMax() *assets.Wei
+	MaxGasCostWei() *assets.Wei
```

Comment:
Let's name this like the others, and move it up to be sorted alphabetically
```suggestion
	CostMax() *assets.Wei
```

Reply:
done